cpp/thirdparty/versions.txt

```diff
@@ -60,7 +60,7 @@ ARROW_LZ4_BUILD_SHA256_CHECKSUM=030644df4611007ff7dc962d981f390361e6c97a34e5cbc3
 # https://github.com/microsoft/mimalloc/issues/353
 ARROW_MIMALLOC_BUILD_VERSION=v1.7.2
 ARROW_MIMALLOC_BUILD_SHA256_CHECKSUM=b1912e354565a4b698410f7583c0f83934a6dbb3ade54ab7ddcb1569320936bd
-ARROW_ORC_BUILD_VERSION=1.6.9
+ARROW_ORC_BUILD_VERSION=1.7.0
 ARROW_ORC_BUILD_SHA256_CHECKSUM=2b79f1152454a38abd7b361b407f65689e275f1fb28d4293892e42fbebcc0a9d
```

Comment:
Hi, @williamhyun .
Could you update this line together?
```
$ shasum -a256 < /Users/dongjoon/APACHE/arrow/cpp/thirdparty/orc-1.7.0.tar.gz
45d6ba9149ffa2aaa168d61ab326f61181861c94529f26da3918a9aa2f801e39
```

FYI, the previous value came from here.
```
$ shasum -a256 < /Users/dongjoon/APACHE/arrow/cpp/thirdparty/orc-1.6.9.tar.gz
2b79f1152454a38abd7b361b407f65689e275f1fb28d4293892e42fbebcc0a9d  -
```